tests/reconfig/setup.md

```diff
@@ -0,0 +1,96 @@
+# Reconfig tests
+
+## Setup
+
+1. Create cloud cluster
+2. Deploy CRDs:
+
+   ```bash
+   kubectl apply -f https://github.com/kubernetes-sigs/gateway-api/releases/download/v0.8.1/standard-install.yaml
+   ```
+
+3. Deploy NGF from edge using Helm install (NOTE: For Test 1, deploy AFTER resources):
+
+   ```bash
+   helm install my-release oci://ghcr.io/nginxinc/charts/nginx-gateway-fabric  --version 0.0.0-edge \
+      --create-namespace --wait -n nginx-gateway
+   ```
+
+4. Optional: Add pod scrape if running in GKE (see [GKE Pod scrape config](#gke-pod-scrape-config)).
```

Comment:
Since we don't have any instructions that leverage GKE Metrics directly, I wonder if it is needed at all

Reply:
Yeah good point. I had there as an alternative to the port-forward, but it's not necessary. I removed it.